scenes/game_elements/props/eternal_loom/components/eternal_loom.gd

```diff
@@ -48,6 +49,9 @@ func _on_interaction_ended() -> void:
 func on_offering_succeeded() -> void:
 	loom_offering_animation_player.play(&"loom_offering")
 	await loom_offering_animation_player.animation_finished
+	#Mark mission completed
+	var quest_path: String = GameState._state.get_value(GameState.QUEST_SECTION, GameState.QUEST_PATH_KEY, "")
+	GameState.add_completed_mission(quest_path)
```

Comment:
This method does not exist in `game_state.gd`. Are you sure that you have committed all of your changes?

Reply:
@AlexanderV24 no need for a different pull request! Just `git add` the missing file and append a commit.